latest_cov_build.py

```diff
@@ -18,3 +33,51 @@
 with tarfile.open(name, 'r:bz2') as tar:
     tar.extractall()
 os.remove(name)
+
+# Geckodriver download
+# OS information for correct geckodriver version
+name = sys.platform
+bitness = platform.architecture()[0]
+latest_version = get_json('https://api.github.com/repos/mozilla/geckodriver/releases/latest')
+tag_name = latest_version['tag_name']
+base_url = 'https://github.com/mozilla/geckodriver/releases/download/' + tag_name
+
+# Linux OS
+if name == 'linux':
```

Comment:
Instead of equality, this should check that the string starts with `linux` (e.g. on my machine sys.platform is `linux2`).